ccpp/data/GFS_typedefs.meta

```diff
@@ -2411,6 +2457,126 @@
+[emdust]
+  standard_name = emission_of_dust_for_smoke
+  long_name = emission of dust for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[emseas]
+  standard_name = emission_of_seas_for_smoke
+  long_name = emission of seas for smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[emanoc]
+  standard_name = emission_of_anoc_for_thompson_mp
+  long_name = emission of anoc for thompson mp
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[ebb_smoke_hr]
+  standard_name = surfce_emission_of_smoke
+  long_name = emission of surface smoke
+  units = ug m-2 s-1
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[frp_hr]
+  standard_name = frp_hourly
+  long_name = hourly frp
+  units = mw
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[frp_std_hr]
+  standard_name = frp_std_hourly
+  long_name = hourly std frp
+  units = mw
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[fhist]
+  standard_name = fhist
+  long_name = fire hist
+  units = none
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[coef_bb_dc]
+  standard_name = coef_bb_dc
+  long_name = coef bb dc from plumerise
+  units = none
+  dimensions = (horizontal_loop_extent)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[ebu_smoke]
+  standard_name = ebu_smoke
+  long_name = smoke buffer of ebu
+  units = various
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[smoke_ext]
+  standard_name = extinction_coefficient_in_air_due_to_smoke
+  long_name = extinction coefficient in air due to smoke
+  units = various
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[dust_ext]
+  standard_name = extinction_coefficient_in_air_due_to_dust
+  long_name = extinction coefficient in air due to dust
+  units = various
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[chem3d]
+  standard_name = chem3d_mynn_pbl_transport
+  long_name = mynn pbl transport of smoke and dust
+  units = various
+  dimensions = (horizontal_loop_extent,vertical_layer_dimension,2)
+  type = real
+  kind = kind_phys
+  active = (flag_for_rrfs_smoke_coupling)
+[min_fplume]
+  standard_name = min_fplume
```

Comment:
plume of what? what is "f" in this case? If it is a height, shouldn't units be meters instead of none, or is it nondimensionalized by some other lengthscale?

Reply:
@grantfirl  plume of fire. "f" represents fire. It is the plumerise maximum/minimum in model layers, so no units.

Reply:
@grantfirl We like this name of minimum_fire_plume_sigma_pressure_level. It is integer in the smoke subroutine. The real type is for diagnostic output purpose.